### ISSUE TITLE
[Angular Universal] mat-datepicker-toggle  at SVGSVGElement.setAttribute - the operation is not allowed by Namespaces in XML\nNAMESPACE_ERR

### ISSUE THREAD

**Issue by patrickmichalina** [angular/material2#7866]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Calendar renders server side

#### What is the current behavior?
using 
```html
<mat-form-field>
      <input matInput [matDatepicker]="picker" placeholder="Date">
      <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
      <mat-datepicker #picker></mat-datepicker>
    </mat-form-field>
```

throws 

```shell
Uncaught (in promise): NAMESPACE_ERR: NAMESPACE_ERR (14): the operation is not allowed by Namespaces in XML\nNAMESPACE_ERR: NAMESPACE_ERR (14): the operation is not allowed by Namespaces in XML\n    at Object.exports.NamespaceError (/Users/Patrick/Repos/myapft/node_modules/domino/lib/utils.js:28:45)\n    at SVGSVGElement.setAttribute 
```

#### What are the steps to reproduce?
Use the above snippet in any universal applications

#### What is the use-case or motivation for changing an existing behavior?
to support server side rendering

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
"@angular/animations": "5.0.0-rc.2",
    "@angular/cdk": "^2.0.0-beta.12",
    "@angular/common": "5.0.0-rc.2",
    "@angular/compiler": "5.0.0-rc.2",
    "@angular/compiler-cli": "5.0.0-rc.2",
    "@angular/core": "5.0.0-rc.2",
    "@angular/flex-layout": "2.0.0-rc.1",
    "@angular/forms": "5.0.0-rc.2",
    "@angular/http": "5.0.0-rc.2",
    "@angular/material": "^2.0.0-beta.12",
    "@angular/platform-browser": "5.0.0-rc.2",
    "@angular/platform-browser-dynamic": "5.0.0-rc.2",
    "@angular/platform-server": "5.0.0-rc.2",
    "@angular/router": "5.0.0-rc.2",
    "@angular/service-worker": "5.0.0-rc.2",

**Comment by crisbeto** [comment]
Sounds like something I ran into in one of the Angular 5 betas. Are you running Angular 5?

**Comment by patrickmichalina** [comment]
@crisbeto yes sorry my versions are:

```json
"@angular/animations": "5.0.0-rc.2",
    "@angular/cdk": "^2.0.0-beta.12",
    "@angular/common": "5.0.0-rc.2",
    "@angular/compiler": "5.0.0-rc.2",
    "@angular/compiler-cli": "5.0.0-rc.2",
    "@angular/core": "5.0.0-rc.2",
    "@angular/flex-layout": "2.0.0-rc.1",
    "@angular/forms": "5.0.0-rc.2",
    "@angular/http": "5.0.0-rc.2",
    "@angular/material": "^2.0.0-beta.12",
    "@angular/platform-browser": "5.0.0-rc.2",
    "@angular/platform-browser-dynamic": "5.0.0-rc.2",
    "@angular/platform-server": "5.0.0-rc.2",
    "@angular/router": "5.0.0-rc.2",
    "@angular/service-worker": "5.0.0-rc.2",
```

**Comment by patrickmichalina** [comment]
My current work around is to NOT use `mat-datepicker-toggle `

**Comment by hamedbaatour** [comment]
Angular 5 is officially released now and I can confirm that this issue is still present with angular universal. Any updates on this?

**Comment by patrickmichalina** [comment]
@hamedbaatour looks like it hasn't been merged in yet

https://github.com/angular/material2/pull/7882

**Comment by hamedbaatour** [comment]
@patrickmichalina never had this issue before with v4. Is nguniveral prohibiting the use of `xmlns` attributes on SVG tags?

**Comment by crisbeto** [comment]
The fix was merged as a part of another PR (#6018).
